mono/mini/mini-exceptions.c

```diff
@@ -726,8 +726,6 @@ unwinder_unwind_frame (Unwinder *unwinder,
 		if (frame->type == FRAME_TYPE_INTERP_TO_MANAGED) {
 			unwinder->in_interp = TRUE;
 			mini_get_interp_callbacks ()->frame_iter_init (&unwinder->interp_iter, frame->interp_exit_data);
-			parent = mini_get_interp_callbacks ()->frame_get_parent (frame->interp_exit_data);
-			MONO_CONTEXT_SET_SP (new_ctx, parent);
```

Comment:
Isn't this needed ?

Reply:
relevant commit: c52e05c19a9982bfa49cd6c7bf8060151a212e4c

[looks like it didn't fix `thread6.exe`](https://github.com/mono/mono/pull/6635/files#r163172823)? 🙂

Reply:
@vargaz usage of the sp in context for the interpreter has been mostly removed here : https://github.com/mono/mono/pull/6644. Since the sp is no longer used for the frame iteration, it no longer contains the InterpFrame which lead to breakage on thread6.exe